examples/scans.py

```diff
@@ -26,8 +27,8 @@ def example(test_name, test_file):
     '''
     scan = client.scan_helper.create(
         name=scan_name,
-        text_targets='tenable.com',
-        template='discovery'
+        text_targets=scan_target,
+        template='basic'
```

Comment:
I am not sure if it's necessary to run `'basic'` here. If `'discovery'` is faster than `'basic'` in anyway, we should keep it as `'discovery'` because it is sufficient for the purpose of examples and not having to add additional time to the already slow test-runs.

Reply:
My only concern was that 'discovery' scans typically only return INFO level vulnerabilities which are not displayed in the Workbench by default. I wasn't sure whether any of the other tests were dependent on data being available there.

Reply:
There is no dependency to this examples script, it is solely for demonstration only. It isn't to test the code, but to test that the examples are not outdated.

Reply:
I'll change back to `'discovery'`, it's much faster than basic.

Reply:
I'm having trouble to get this to pass in our engineering environment with the template being set to `'discovery'`, for a single target without auth, a basic scan really shouldn't take too much longer.